### DISCUSSION TITLE
[v2]How to set the hiddenTitle property in rust

### DISCUSSION THREAD

**Discussion by nongyehong** [tauri-apps/tauri#11040] [Q&A] [answered]
![2d9a63fd369438457e79732497995098](https://github.com/user-attachments/assets/c842c6ff-fa45-4bb6-a90f-dbc713ea53ab)
![008389ba0dbda8f4e1a915d9a3029a81](https://github.com/user-attachments/assets/3287b929-5d39-4f40-b8e3-eb378bc119b6)
I'm a newbie to Rust and I'd like to get get_webview_window in the method above to set the hiddenTitle property and the visible property, but I've tried to have no of them in it, so where should I get them

**Comment by FabianLars** [comment]
The title visibility status can only be set when [creating the window](https://docs.rs/tauri/2.0.0-rc/x86_64-apple-darwin/tauri/webview/struct.WebviewWindowBuilder.html#method.hidden_title) and can't be changed later (not sure if technical limitation or just a missing implementation on our side).

To confirm the last part, you could add the objc and cocoa crates to your app and do something like this (untested and written here so may be incomplete)
```rs
let ns_window = window.ns_window().unwrap() as *mut Object;
ns_window.setTitleVisibility_(appkit::NSWindowTitleVisibility::NSWindowTitleVisible);
```

**Comment by nongyehong** [comment, marked as answer]
I found a solution, it turns out that it can be configured separately, and I need this configuration so that I don't need to change the style of the window after I start the project, which is mentioned in the documentation: https://v2.tauri.app/develop/configuration-files/#platform-specific-configuration
